indexer/indexer_tests/trie_test.cpp

```diff
@@ -83,74 +84,82 @@ string DebugPrint(KeyValuePair const & p)
 
 struct KeyValuePairBackInserter
 {
-  vector<KeyValuePair> m_v;
   template <class TString>
-  void operator()(TString const & s, trie::FixedSizeValueReader<4>::ValueType const & rawValue)
+  void operator()(TString const & s, uint32_t const & value)
   {
-    uint32_t value;
-    memcpy(&value, &rawValue, 4);
     m_v.push_back(KeyValuePair(s, value));
   }
+
+  vector<KeyValuePair> m_v;
 };
 
-struct MaxValueCalc
+// The SingleValueSerializer and ValueList classes are similar to
+// those in indexer/string_file_values.hpp but that file
+// is not included to avoid coding_tests's dependency from indexer.
```

Comment:
Oh, the comment is wrong now. These are indexer_tests not coding_tests.
I removed this part of the comment.
I still don't think we should include string_file_values.hpp here.